src/dlgProfilePreferences.cpp

```diff
@@ -1626,6 +1631,29 @@ void dlgProfilePreferences::copyMap()
     mudlet::self()->setShowMapAuditErrors(savedOldAuditErrorsToConsoleEnabledSetting);
 }
 
+void dlgProfilePreferences::slot_setLogDir()
+{
+    Host* pHost = mpHost;
+    if (!pHost) {
+        return;
+    }
+
+    QDir currentLogDir = QFileDialog::getExistingDirectory(this, tr("Select log directory"), logDirPath, QFileDialog::ShowDirsOnly);
+    logDirPath = currentLogDir.absolutePath();
+    return;
+}
+
+void dlgProfilePreferences::slot_resetLogDir()
+{
+    Host* pHost = mpHost;
+    if (!pHost) {
+        return;
+    }
+
+    logDirPath = "";
```

Comment:
This is a `QString` so `logDirPath.clear();` would better broadcast your intent! {And avoid a raw string conversion error when we *eventually* do a `#define QT_NO_CAST_FROM_ASCII` & `#define QT_NO_CAST_TO_ASCII` when we have to play *go hunt the strings that need translation* for Mudlet 4.0 ...) :nerd_face:

Reply:
Oops. Brain fart.